api/validation.go

```diff
@@ -28,45 +29,68 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/labstack/echo/v4"
-	"github.com/pkg/errors"
+	everestv1alpha1 "github.com/percona/everest-operator/api/v1alpha1"
 	"go.uber.org/zap"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
 
 	"github.com/percona/percona-everest-backend/cmd/config"
 	"github.com/percona/percona-everest-backend/model"
 )
 
+const (
+	pxcDeploymentName   = "percona-xtradb-cluster-operator"
+	psmdbDeploymentName = "percona-server-mongodb-operator"
+	pgDeploymentName    = "percona-postgresql-operator"
+)
+
 var (
-	errDBCEmptyMetadata   = errors.New("DatabaseCluster's Metadata should not be empty")
-	errDBCNameEmpty       = errors.New("DatabaseCluster's metadata.name should not be empty")
-	errDBCNameWrongFormat = errors.New("DatabaseCluster's metadata.name should be a string")
+	errDBCEmptyMetadata      = errors.New("DatabaseCluster's Metadata should not be empty")
+	errDBCNameEmpty          = errors.New("DatabaseCluster's metadata.name should not be empty")
+	errDBCNameWrongFormat    = errors.New("DatabaseCluster's metadata.name should be a string")
+	errNotEnoughMemory       = errors.New("Memory limits should be above 512M")                                                             //nolint:stylecheck
+	errInt64NotSupported     = errors.New("Specifying resources using int64 data type is not supported. Please use string format for that") //nolint:stylecheck
+	errNotEnoughCPU          = errors.New("CPU limits should be above 600m")                                                                //nolint:stylecheck
+	errNotEnoughDiskSize     = errors.New("Storage size should be above 1G")                                                                //nolint:stylecheck
+	errUnsupportedPXCProxy   = errors.New("You can use only either HAProxy or Proxy SQL for PXC clusters")                                  //nolint:stylecheck
```

Comment:
```suggestion
	errUnsupportedPXCProxy   = errors.New("You can use either HAProxy or Proxy SQL for PXC clusters")                                  //nolint:stylecheck
```

What is the name we use in UI for PXC? Shouldn't we use it instead of PXC?